tests/test_plugin_features.py

```diff
@@ -0,0 +1,136 @@
+import pathlib
+import numpy as np
+import pytest
+
+import dclab
+from dclab.rtdc_dataset.plugins.plugin_feature import (
+    PlugInFeature, find_plugin_feature_script, remove_all_plugin_features)
+
+from helper_methods import retrieve_data
+
+
+@pytest.fixture(autouse=True)
+def cleanup_plugin_features():
+    """Fixture used to setup and cleanup some fake ancillary features"""
+    # code run before the test
+    pass
+    # then the test is run
+    yield
+    # code run after the test
+    # remove our test plugin examples
+    remove_all_plugin_features()
+
+
+def get_plugin_file(plugin_name="plugin_test_example.py"):
```

Comment:
To be consistent with the other tests, I would remove this function and use `data_dir=pathlib.Path(...)` at the top of the file and in the test function I would then use`plugin_path = data_dir / "plugin_test_example.py"`.

Reply:
Cool, will do. I thought it'd be a bit cleaner this way, similar to what you had done in `test_isoelastics.py`:
```python
def get_isofile(name="example_isoelastics.txt"):
    thisdir = pathlib.Path(__file__).parent
    return thisdir / "data" / name
```

If we want to be consistent, shall I change these kinds of functions everywhere to the `data_dir=...` code?